content/turbo fish.md

```diff
@@ -0,0 +1,7 @@
++++
+title = "Turbo fish"
```

Comment:
Also regarding consistent casing, this one has a similar nature to the toilet closure (we usually do not capitalize either one), and so they should follow the same capitalization rules.